app/views/coronavirus_landing_page/components/landing_page/_nhs_notice.html.erb

```diff
@@ -0,0 +1,45 @@
+<%
+  # hardcoding data that is expected to be in the Content Item
+  nhs_banner = {
+    "sections" => [
+      {
+        "heading" => "If you have no symptoms:",
+        "markdown" => "Check if you should get [regular rapid lateral flow tests](/getting-tested-for-coronavirus).",
+      },
+      {
+        "heading" => "If you have coronavirus symptoms:",
+        "markdown" => "- [get a PCR test](/get-coronavirus-test)\n- stay at home",
+      }
+    ]
+  }
+%>
```

Comment:
will this be removed before merging?

Reply:
No, it will be removed after deployment when I've updated the YAML file.